protocol/v2/ssv/validator/committee.go

```diff
@@ -293,6 +316,41 @@ func (c *Committee) ProcessMessage(logger *zap.Logger, msg *queue.SSVMessage) er
 	return nil
 
 }
+func (c *Committee) StopOldRunners(logger *zap.Logger, currentSlot phase0.Slot) error {
+	if maxRunnerAgeSlot > currentSlot {
+		return nil
+	}
+
+	firstTooOldSlot := currentSlot - maxRunnerAgeSlot
+
+	c.mtx.Lock()
+	defer c.mtx.Unlock()
+
+	for slot := range c.Runners {
+		if slot <= firstTooOldSlot {
+			logger.Info("stopping old committee runner", zap.Uint64("slot", uint64(slot)))
+			delete(c.Runners, slot)
```

Comment:
dont we also need to delete the queue?

Reply:
@moshe-blox Actually, yep, good catch. We need to delete queue even we stoped the consumer of it.